ocs_ci/utility/utils.py

```diff
@@ -2642,13 +2642,14 @@ def dump_config_to_file(file_path):
         yaml.safe_dump(config_copy, fs)
 
 
-def create_rhelpod(namespace, pod_name, timeout=300):
+def create_rhelpod(namespace, pod_name, rhel_version=8, timeout=300):
```

Comment:
With this change, it has to be fixed also the usage of `create_rhelpod` with `timeout` parameter in those two places (hopefully I didn't miss anything):
* https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/platform_nodes.py#L971
```python
        rhel_pod_obj = create_rhelpod(constants.DEFAULT_NAMESPACE, rhel_pod_name, 600)
```
* https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/install_ocp_on_rhel.py#L68
```
        self.rhelpod = create_rhelpod(constants.DEFAULT_NAMESPACE, self.pod_name, 600)
```

Reply:
done